airflow/decorators/__init__.pyi

```diff
@@ -245,5 +247,11 @@ class TaskDecoratorCollection:
         :param cap_add: Include container capabilities
         """
         # [END decorator_signature]
+    @overload
+    def sensor(self, *, python_callable: Optional[Callable] = None, **kwargs) -> TaskDecorator:
+        """
+        Wraps a Python function into a sensor operator.
+        :param python_callable: decorated function that implements the poke() logics of a sensor operator.
+        """
```

Comment:
`python_callable` can’t be keyword-only, it needs to be accepted as positional for the decorator to work. Also individual arguments should be spelled out explicitly (instead of using `kwargs`) since this file is providing editor autocompletion, and `kwargs` is mostly useless for autocompletion.

Reply:
The only argument useful for the sensor decorator is "python_callable", but please let me know if there are other arguments that need to be explicit.

Reply:
By not using `PythonSensor`, this implementation looses the ability to pass context variables (such as `ti`) to the callable. The ability of passing in `templates_dict` is also lost.